### PR TITLE
Adds terraform example

### PR DESCRIPTION
**Pull request by pollosp**
Hello, 
I think a terraform example could be useful for this project. I think it makes easier your lambdas deployment.

### CODE REVIEW

**Comment by ajlanghorn** [comment]
+1 to having this merged, save me building my own worked example out.

File: terraform/example/Makefile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	make -C ../../ archive
+init:
+	docker run --rm -ti \
+		-v $(current_dir):$(container_dir) \
+		-e ENVIRONMENT=$(ENVIRONMENT) \
+		-e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) \
+		-e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY)\
+		-e AWS_DEFAULT_REGION=$(AWS_DEFAULT_REGION) \
+		--workdir=$(container_dir)\
+		hashicorp/terraform:0.11.1 \
```
Comment (ajlanghorn) [inline terraform/example/Makefile]: Can we pull `hashicorp/terraform:latest` instead, and not pin the version in the Terraform provider instantiation? That's less overhead in future.

Comment (pravsingh) [reply terraform/example/Makefile]: we should fix to a latest known stable version. Make it property driven so that it can be upgraded easily in future.

**Comment by jaygorrell** [comment]
Apologies for the delay here. I think parameterizing the TF version would be good, as well as a quick pass through `terraform fmt`

**Comment by chrisgilmerproj** [comment]
I have built a project for this in terraform: https://github.com/trussworks/terraform-aws-s3-anti-virus

You can use it from the registry here: https://registry.terraform.io/modules/trussworks/s3-anti-virus

Maybe we can update the docs to point to it.

**Pull request closed without merging**